### ISSUE TITLE
Screenreader does not read found set size in Quickpick

### ISSUE THREAD

**Issue by weinand** [microsoft/vscode#93941]
Refs #93648 
Orca 3.28.0 on Ubuntu 18.04.4 LTS

- open file quickpick
- type a pattern into quickpick's text box until the found set shows only one item 

Screen reader stays silent.
I would expect that at least the number of found items is read.

[I have no experience with screen readers. I just report what I noticed.
Someone who has to rely on a screen reader should test this and provide feedback.]

**Comment by bpasero** [comment]
@weinand oh, invalid? can you calrify?

**Comment by weinand** [comment]
With Orca 3.37.1pre the number of results is read correctly.

**Comment by weinand** [comment]
@bpasero the correct version of Orca was not specified anyhwere... I was using 3.28.0.
